### DISCUSSION TITLE
apisix支持国产操作系统吗，像麒麟，欧拉，统信，中科方德等？

### DISCUSSION THREAD

**Discussion by xingdong123** [apache/apisix#7714] [General]
apisix支持国产操作系统吗，像麒麟，欧拉，统信，中科方德等？
 如果支持，如何部署？谢谢！

**Comment by spacewander** [comment]
We have some users who are using these OS.
As for the deployment, it depends on the corresponding system environment.

**Reply by xingdong123** [reply]
Are there corresponding documents or materials?
There are only CentOS and Ubuntu on the official website.

**Reply by xingdong123** [reply]
信创用的是国产操作系统，官方后面是否会支持国产系统？

**Reply by tzssangglass** [reply]
You can try to compile it yourself, APISIX compiles fine on arm64.

**Reply by spacewander** [reply]
You can use and modify the scripts in https://github.com/api7/apisix-build-tools to build the rpm/deb for the corresponding OS. AFAIK, these OS have good support of rpm/deb. But you may need to do some modifications as they are not 100% compatible.

**Comment by moonming** [comment]
![image](https://user-images.githubusercontent.com/26448043/185526376-ecee80af-a4a6-4725-af7d-2e54c10fbac6.png)

more certifications in https://www.apiseven.com/
